facefusion/core.py

```diff
@@ -71,6 +71,7 @@ def cli() -> None:
 	group_face_mask.add_argument('--face-mask-types', help = wording.get('face_mask_types_help'), dest = 'face_mask_types', default = [ 'box' ], choices = facefusion.choices.face_mask_types, nargs = '+')
 	group_face_mask.add_argument('--face-mask-blur', help = wording.get('face_mask_blur_help'), dest = 'face_mask_blur', type = float, default = 0.3, choices = facefusion.choices.face_mask_blur_range, metavar = create_metavar(facefusion.choices.face_mask_blur_range))
 	group_face_mask.add_argument('--face-mask-padding', help = wording.get('face_mask_padding_help'), dest = 'face_mask_padding', type = int, default = [ 0, 0, 0, 0 ], nargs = '+')
+	group_face_mask.add_argument('--face-mask-parser-regions', help = wording.get('face_mask_parser_regions_help'), dest = 'face_mask_parser_regions', default = [ 'skin', 'left-eyebrow', 'right-eyebrow', 'left-eye', 'right-eye', 'nose', 'upper-lip', 'lower-lip' ], choices = facefusion.choices.face_mask_parser_regions, nargs = '+')
```

Comment:
Just `--face-mask-regions` ... can we have `all` for default? which is an alias to enable them all...